.github/workflows/tests.yml

```diff
@@ -0,0 +1,78 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+env:
+  DRIVER_URL: "http://localhost:4444/wd/hub"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+
+  tests:
+    name: Tests
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php: [ 5.4, 5.5, 5.6, 7.0, 7.1, 7.2, 7.3, 7.4, 8.0, 8.1 ]
+      fail-fast: false
+    env:
+      MATRIX_PHP: ${{ matrix.php }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          coverage: "none"
+          php-version: "${{ matrix.php }}"
+          tools: composer
+
+      - name: Configure for PHP >= 8.0
+        if: "${{ matrix.php >= '8.0' }}"
+        run: |
+          composer config platform.php 7.4.99
```

Comment:
Why do we need this ?

Reply:
It was in the travis file. Let's remove it.